frappe/contacts/doctype/address/address.py

```diff
@@ -59,6 +60,10 @@ def validate_reference(self):
 			to_remove = [row for row in self.links if row.link_doctype != "Company"]
 			[ self.remove(row) for row in to_remove ]
 
+	def make_complete_address(self):
+		# for next line in link field options
+		self.complete_address = "<br>" + self.get_display()
```

Comment:
why `<br>` in start?

Reply:
Used the <code>br</code> tag so that the address can be seen on newline of link field options, aimed at making it readable. Tried to see if it can be handled in the link control where this description for each item is made in html but to add it specifically before address field involved extra lines of code with different cases like an absent search_field.

1. Options with a <code>br</code>
![with_br](https://user-images.githubusercontent.com/25682214/35621336-0436e054-06aa-11e8-810d-99afa39a814d.png)
 
2. Options without a <code>br</code>
![without_br](https://user-images.githubusercontent.com/25682214/35621347-0b63ce46-06aa-11e8-94d9-48e968765ce0.png)


Also, I've changed the ERPNext's referred PR and its no more dependent on this PR, but the decision of a <code>br</code> in field versus a change in the way a link control's html is made and to introduce a <code>br</code> there is still to be made. Haven't figured out the ideal solution for it yet.